<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZooCmdUIClass</class>
 <widget class="QMainWindow" name="ZooCmdUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zooCmdUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="zooCmdUI.qrc">
    <normaloff>:/images/Resources/images/app.png</normaloff>:/images/Resources/images/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_sim"/>
   <addaction name="action_setup"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_setup">
   <property name="icon">
    <iconset resource="zooCmdUI.qrc">
     <normaloff>:/images/Resources/images/setup.png</normaloff>:/images/Resources/images/setup.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="action_sim">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="zooCmdUI.qrc">
     <normaloff>:/images/Resources/images/start.png</normaloff>
     <normalon>:/images/Resources/images/stop.png</normalon>:/images/Resources/images/start.png</iconset>
   </property>
   <property name="text">
    <string>仿真</string>
   </property>
   <property name="toolTip">
    <string>仿真</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="zooCmdUI.qrc">
     <normaloff>:/images/Resources/images/open.png</normaloff>:/images/Resources/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="zooCmdUI.qrc">
     <normaloff>:/images/Resources/images/save.png</normaloff>:/images/Resources/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="zooCmdUI.qrc"/>
 </resources>
 <connections/>
</ui>
